packages/aws-cdk-lib/aws-elasticloadbalancingv2/lib/alb/application-load-balancer.ts

```diff
@@ -119,6 +119,7 @@ export class ApplicationLoadBalancer extends BaseLoadBalancer implements IApplic
     if (props.idleTimeout !== undefined) { this.setAttribute('idle_timeout.timeout_seconds', props.idleTimeout.toSeconds().toString()); }
     if (props.dropInvalidHeaderFields) {this.setAttribute('routing.http.drop_invalid_header_fields.enabled', 'true'); }
     if (props.desyncMitigationMode !== undefined) {this.setAttribute('routing.http.desync_mitigation_mode', props.desyncMitigationMode); }
+    if (props.deletionProtection !== undefined) { this.setAttribute('deletion_protection.enabled', props.deletionProtection.toString()); }
```

Comment:
This seems [redundant](https://github.com/aws/aws-cdk/blob/28c4be3419e8c18ac84f2b72ae1838f8f424c1c6/packages/aws-cdk-lib/aws-elasticloadbalancingv2/lib/shared/base-load-balancer.ts#L231) 🤔

Reply:
Oops.. 
I have removed this line. Thanks!